config/mcp-system/broker-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
```

Comment:
already exists no?

https://github.com/kagenti/mcp-gateway/blob/main/config/mcp-system/service.yaml#L3

Reply:
The existing service https://github.com/kagenti/mcp-gateway/blob/main/config/mcp-system/service.yaml#L6 already serves both the broker (port 80->8080) and router (50051). The HTTPRoute in this PR could just reference mcp-gateway on port 80 instead of creating a new mcp-broker service, similar to how   https://github.com/kagenti/mcp-gateway/blob/main/config/istio/gateway/httproute.yaml#L17-L18 does it. does that make sense?